### ISSUE TITLE
Parse error

### ISSUE THREAD

**Issue by ese942** [brython-dev/brython#1555]
There seems to be an error in brython parser.  It's triggered by comments in multiline expressions.

This code works fine with python 2 and 3 but not with brython:
```
data = {
    'func' : lambda x : false and
        false or   # option 1
        x               # option 2
}
data ['func'] ('foo')
data ['func'] (False)
```

### Expected result
foo
False

### Brython result
```
handle error {__class__: {…}, __mro__: Array(3), $is_class: true, $infos: {…}, $factory: ƒ, …} (5) ["invalid syntax", "__main__", 2, 67, "return  false and         false or   # option 1       x          # option 2 ", __true, __class__: {…}]
Traceback (most recent call last):
  File __main__, line 2
    return  false and       false or   # option 1       x          # option 2
SyntaxError: invalid syntax
```

The same multiline expression fails outside a dict in python 2/3 due to line termination.  But it's allowed inside a dict.

The expression works fine in brython without the comments.

The lambda isn't necessary to trigger the bug.  It's just part of my setup.  It's in the example to validate that the fix covers this case.

Thanks, hope this helps!
Edward

**Comment by PierreQuentel** [comment]
@ese942 
You are probably using the latest release, because in the current development version the bug is fixed, as a consequence of commit #0686d11efa5df11bd283c088521039982e090768 which fixed issue #1545

**Comment by ese942** [comment]
Thanks, got it.  I reproduced it in the brython.info console which runs
3.9.  Didn't try the dev version.
Sorry for the duplication.
